docs/api/navigation-history.md

```diff
@@ -65,3 +62,24 @@ Navigates to the specified offset from the current entry.
 #### `navigationHistory.length()`
 
 Returns `Integer` - History length.
+
+#### `navigationHistory.deleteEntryAtIndex(index)`
+
+* `index` Integer
+
+Deletes the navigation entry at the given index. Can't delete entry at the "current active index".
+
+Returns `boolean` - Whether the navigation entry was removed from the webContents history.
+
+#### `navigationHistory.get()`
```

Comment:
Sorry to go back and forth on this, but I actually think naming here is non-idiomatic for an Electron API. Typically we use the convention of `.getAll*()`. This can be seen with `webContents.getAllWebContents()`, `session.getAllExtensions()`, and `BaseWindow.getAllWindows()`.

`session.cookies.get()` exists, but supports filtering options which this API likely won't have.

Given this, I think `getAllEntries()` or `getAll()` would be preferred here.

Reply:
Nice call out! Given the naming of the other APIs, `getAllEntries()` is not too redundant.


Naming is always the hardest part of the job haha 😅

Reply:
I renamed the method to getAllEntries().